src/backend/api/handlers.py

```diff
@@ -0,0 +1,191 @@
+# coding=utf-8
+import math
+import os
+from typing import AnyStr
+
+import dateparser
+
+from api.exceptions import BadRequestFormatExceptioin
+from api.mirrors_update import (
+    get_config,
+    get_verified_mirrors,
+    REQUIRED_MIRROR_PROTOCOLS,
+)
+from db.db_engine import GeoIPEngine
+from db.models import Url, Mirror
+from db.utils import session_scope
+from sqlalchemy.sql.expression import false
+
+from common.sentry import (
+    get_logger,
+)
+
+
+MAX_LENGTH_OF_MIRRORS_LIST = 5
+
+logger = get_logger(__name__)
+
+
+def _get_distance_between(
+        lat1: float,
+        lon1: float,
+        lat2: float,
+        lon2: float,
+) -> float:
+    return math.sqrt(
```

Comment:
Just as a little tip: sqrt is quite expensive operation and you don't actually need the distance itself to sort mirrors by distance. So you can easily drop that math.sqrt call and return squared value and sort by it.

Reply:
Done. It's replaced by hybrid method in the model